src/components/AvatarWithIndicator.tsx

```diff
@@ -1,45 +1,41 @@
 import React from 'react';
 import {View} from 'react-native';
-import PropTypes from 'prop-types';
-import Avatar from './Avatar';
-import styles from '../styles/styles';
-import Tooltip from './Tooltip';
+import {SvgProps} from 'react-native-svg';
 import * as UserUtils from '../libs/UserUtils';
-import Indicator from './Indicator';
+import styles from '../styles/styles';
+import Avatar from './Avatar';
 import * as Expensicons from './Icon/Expensicons';
+import Indicator from './Indicator';
+import Tooltip from './Tooltip';
 
-const propTypes = {
+type AvatarWithIndicatorProps = {
     /** URL for the avatar */
-    source: PropTypes.oneOfType([PropTypes.string, PropTypes.func]).isRequired,
+    source: string | React.FC<SvgProps>;
```

Comment:
Maybe we should create this as a seperate type? 
I think I am also using it in `ReportUtils`

Reply:
@kubabutkiewicz Is it related to "avatar source" where you are using in `ReportUtils`?

Reply:
Yes